app/layout.js

```diff
@@ -0,0 +1,9 @@
+export default function RootLayout({ children }) {
+    return (
+      <html>
+        <head />
```

Comment:
Can we add some meta tags like title here?

Reply:
It looks like that meta is also required to be able to support CSS media queries : https://github.com/vercel/next.js/issues/41842#issuecomment-1292104770